### ISSUE TITLE
I just switched to shinydashboard, and I can not see the shiny-server log-out button anymore

### ISSUE THREAD

**Issue by zhongxiali** [rstudio/shinydashboard#77]
I just switched to shinydashboard, and I can not see the shiny-server log-out button anymore.
I checked the HTML code, it is there, just not showing.
this is really bad, I have to switch back, if this can not be fixed,

<div class="shiny-server-account">  Logged in as <span class="shiny-server-username">staff</span>  <a href="__logout__">Logout</a></div>

**Comment by YvesCR** [comment]
I had the issue as well.

I fixed it thanks to the support of rstudio.
This is their solution:

There are multiple ways you can get around this problem by changing your layout. But may be the best (and more elegant way) would be to change the shiny-server.css file for that specific element. 
The css file is located here:
/opt/shiny-server/assets/shiny-server.css
Edit this file. In the section ".shiny-server-account", add a "z-index" with a large value. I have tried it with value of 2000, as in the following:
```javascript
.shiny-server-account {
position: absolute;
top: 0;
right: 0;
width: auto;
height: auto;
background-color: #E0E0E0;
padding: 3px 8px;
font-size: 12px;
z-index: 2000;
}
```

It worked for me,

best

**Comment by haozhu233** [comment]
Is that possible that Rstudio set it as a default? The logout button will be hidden under the navbar in a navbarPage as well. It is a reasonable feature that in most cases, people would want the logout button float on the top of everything.

**Comment by wch** [comment]
What version of shinydashboard are you using? That should be fixed in version 0.5.1, via this commit: 682e12c. Also, see: http://rstudio.github.io/shinydashboard/appearance.html#logout-panel

**Comment by haozhu233** [comment]
@wch Hi, Wilson, I wan't using shinydashboard. Instead, I was using a regular `navbarPage` page in basic shiny plus a Bootstrap style sheet. Sorry for the off-topic comment under this repo.

I met the same issue when I was trying to deploy that app to a server with the latest shiny-server pro installed. I found this post when I was trying to do a search online and fixed this issue by adjusting the z-index in the shiny-server. 

I think this problem can be permanently fixed by changing the shiny-server.css in shiny-server package (or close-sourced shiny-server Pro).
